docs/product-installation-guide/src/main/asciidoc/clustering-dev-proc.adoc

```diff
@@ -1,17 +1,17 @@
 [id='clustering-dev-proc']
 = Clustering {PRODUCT} for a development environment
 
-Developers use {PRODUCT} to create rules and processes that users then run for a runtime environment to assist with decision making. The primary benefit of clustering {PRODUCT} development environments is high availability and enhanced collaboration. For example, if clustering is configured and Developer X is authoring rules on {CENTRAL} Node1 then Developer Y, who is working on {CENTRAL} Node2, can see the rules authored by Developer X as they are created. Furthermore, all of the developers working on individual nodes of the cluster will see the same synchronized content. In addition, clustering provides high availability in the rules development environment. If Developer X is working on Node1 and that node fails, Developer X's work is preserved and visible on any other node of the cluster.
+Developers use {PRODUCT} to create rules and processes that users then run to assist with decision making. The primary benefit of clustering {PRODUCT} development environments is high availability and enhanced collaboration. For example, if clustering is configured and Developer X is authoring rules on {CENTRAL} Node1 then Developer Y, who is working on {CENTRAL} Node2, can see the rules authored by Developer X as they are created. Furthermore, all of the developers working on individual nodes of the cluster will see the same synchronized content. In addition, clustering provides high availability in the rules development environment. If Developer X is working on Node1 and that node fails, Developer X's work is preserved and visible on any other node of the cluster.
```

Comment:
This is a good time to get your thoughts on a recurring terminology issue. You don't have to change anything if you don't want, but wanted to loop you in.

A recurring terminology issue is what to call the "stuff" that users create in Central. The blanket term to date has been "business assets" and "assets", which encompass rules, processes, tasks, etc. Specifying "rules and processes" is tricky because it's a reuse issue (doesn't apply to DM, which has only rules). What I've found helpful is to subtly define assets in passing the first time, then use assets when referring collectively after that. Christy and I have been in discussion about this a lot and something I've been meaning to ask you about. 

So here, it could be "to create business assets, such as rules, processes, and tasks, that users then run..." This first mention could be ifdefed, then subsquent mentions could just be to "assets" or "business assets".

Now, this poses a problem: "assets" is vague. Assets are what these things are called in Central, yes, but everything else is also called assets (processes, tasks, decision tables, drl files, all alike). So when you execute an "asset", you're really executing rules, processes, or tasks (I think that's it, maybe more). To make things more complicated, what is a blanket term for complete "decision services + process applications + task lists"? These are the finished products of the "assets" and are what we are also executed as a whole.

In short, these are the terms I had to come up with in the Packaging doc with no alternative and that we should decide on as a team soon (post GA):
- rules, processes, tasks, decision tables, data objects, bla bla = "business assets" / "assets"
- decision services, process applications, task lists = "services"

The "services" remedy is not great and SMEs only let me use that if I define it upon first mention, and even then they're not totally sold on it but don't have suggestions. I don't know what else to do and would love your input before asking the team. Maybe a call sometime.